### PR TITLE
Submission title length now must be greater than 20

### PR DESCRIPTION
**Pull request by yesiamquitesmall**
This should make the submission title length have to be over 20 characters. Was this what #345 was asking for?

### CODE REVIEW

**Comment by voat** [comment]
Hi and thank you for this PR. You are correct, this is what #345 was about. There is one small issue with your PR though. You are rejecting all submissions which have titles <= 20, meaning that titles which have exactly 20 characters will also get rejected ;)

To make #345 fully implemented, changes also need to be made to the following files:

https://github.com/voat/voat/blob/master/Whoaverse/Whoaverse/Models/AddSubmissionLinkpost.cs https://github.com/voat/voat/blob/master/Whoaverse/Whoaverse/Models/AddSubmissionSelfpost.cs

**Comment by yesiamquitesmall** [comment]
I think that should do it, I'm sorry it took so long, this is my first time using git hub properly.

**Comment by voat** [comment]
Looks good, thanks! I'll merge it now but it will not go live immediately, it may take a few days. Keep the PRs coming :+1:

File: Whoaverse/Whoaverse/Models/AddSubmissionSelfpost.cs [hunk 1]
```diff
@@ -34,8 +34,8 @@ public class AddSubmissionSelfpost
 
 
         [Required(ErrorMessage = "Message title is required. Please fill this field.")]
-        [StringLength(200, ErrorMessage = "Submission title is limited to 200 characters.")]
-        public string Title { get; set; }
+        [StringLength(200, ErrorMessage = "Submission title must be at least 20 and is limited to 200 characters long.", MinimumLength = 20)]
+        public string Linkdescription { get; set; }
```
Comment (quantumwannabe) [inline Whoaverse/Whoaverse/Models/AddSubmissionSelfpost.cs]: It looks like there was a getter/setter for "Title", but now there is one for "Linkedescription" instead. Is this intended?

Comment (yesiamquitesmall) [inline Whoaverse/Whoaverse/Models/AddSubmissionSelfpost.cs]: Nope it wasn't, I seem to be creating more problems than I'm fixing :L

**Pull request merged**